docs/release-guide.md

```diff
@@ -1,36 +1,79 @@
 # Release Guide
 
-## Container Image
-
-### Semi-automatic
```

Comment:
the semi-automatic and manual image pushing steps were all the same except for the command to manually push, so I removed these sections and added a note that it can be manually pushed instead